html/modules/custom/iasc_content/iasc_content.module

```diff
@@ -1245,6 +1252,29 @@ function iasc_content_preprocess_node(&$vars) {
     }
   }
 
+  // Check for exclude on archive.
+  foreach ($activGroupListEntity as $groupContent) {
```

Comment:
This variable name is already used in other places in the module file, but to me it would better with an extra 'e' - `activeGroupListEntity`